packages/zoe/test/swingsetTests/zoe-metering/infiniteTestLoop.js

```diff
@@ -1,15 +1,15 @@
 import harden from '@agoric/harden';
 
-export const makeContract = zoe => {
-  const invite = zoe.makeInvitation(() => {}, 'tester');
-  return harden({
-    invite,
-    publicAPI: {
+export const makeContract = zcf => {
+  const invite = zcf.makeInvitation(() => {}, 'tester');
```

Comment:
This isn't needed until the return, so just return it there. Unless the test needs it to be made before doTest() is defined?

Reply:
I think this is a matter of taste, but this aspect hasn't changed since Michael wrote it. I think I like explicitly creating the invite better, if I had to chose, so I would go with how it was originally written.